guides/Dataset.md

```diff
@@ -282,6 +283,41 @@ NOTE: If you have a DOI, the citation text can be [automatically generated](http
 }
 </pre>
 
+#### How to reference Short DOIs
+
+[Short DOIs](http://shortdoi.org/) is a redirect service offered by the International DOI Foundation that provides a shorter version of an orginial DOI. For example, the original DOI `doi:10.5066/F7VX0DMQ` has a short DOI of `doi.org/csgf`. Short DOIs are resolvable using standard DOI URLS such as `http://doi.org/fg5v`. These short DOIs are treated identically to the original DOI. If you are using the short DOI service, we recommend publishing a short DOI URL using the `schema:sameAs` property of the `schema:PropertyValue` which is describing the original DOI:
+
+<pre>
+{
+  "@context": {
+    "@vocab": "https://schema.org/"
+  },
+  "@type": "Dataset",
+  "name": "Removal of organic carbon by natural bacterioplankton communities as a function of pCO2 from laboratory experiments between 2012 and 2016",
+  "description": "This dataset includes results of laboratory experiments which measured dissolved organic carbon (DOC) usage by natural bacteria in seawater at different pCO2 levels. Included in this dataset are; bacterial abundance, total organic carbon (TOC), what DOC was added to the experiment, target pCO2 level. ",
+  "url": "https://www.sample-data-repository.org/dataset/472032",
+  "sameAs": "https://search.dataone.org/#view/https://www.sample-data-repository.org/dataset/472032",
+  "version": "2013-11-21",
+  "keywords": ["ocean acidification", "Dissolved Organic Carbon", "bacterioplankton respiration", "pCO2", "carbon dioxide", "oceans"],
+  "identifier":
+      {
+        "@id": "https://doi.org/10.5066/F7VX0DMQ",
+        "@type": "PropertyValue",
+        "propertyID": "https://registry.identifiers.org/registry/doi",
+        "value": "doi:10.5066/F7VX0DMQ",
+        "url": "https://doi.org/10.5066/F7VX0DMQ",
+	<strong>"sameAs: "http://doi.org/fg5v"</strong>
+      }
+}
+</pre>
+
+`schema:sameAs` is used here for the following reasons:
+
+1. It doesn't add too many more statements that might the page weight (which may impact major search engine crawlers stopping the crawl of schema.org markup).
```

Comment:
should be: "that might **increase** the page weight..."

Reply:
ah yeah good question. My first thought was that since shortDOI is dependent on the original DOI, the sameAs should go on the DOI directly since its an identifier. Otherwise, why not just put all identifier URIs in the `sameAs` on the Dataset? And maybe we should do that?

But happy to update and switch to the Dataset.

Reply:
I think having them in one place (all as `Dataset` properties) makes the whole thing easier to process. Otherwise we have a `sameAs` that makes an instance of `Identifier` equivalent to an instance of `Dataset`.

Reply:
Isn't the short DOI an Identifier, not a Dataset?

Reply:
ah, i see what you are saying. just resolved the schema.org fo rthe short DOI again, and it does come back types as Dataset. ok, cool. i'll change it here. sorry!